frontend/src/app/modules/auth/register-form/register-form.component.html

```diff
@@ -2,46 +2,90 @@
     <img src="../../../../assets/icons/logo.svg" alt="logo" />
     <div class="to-login-container">
         <h2>Create an account</h2>
-        <p class="already-registered">Already registered? <span>Log in</span></p>
+        <p class="already-registered">
+            Already registered? 
+            <button routerLink="/login">
+                <span> Log in</span>
+            </button>
+        </p>
     </div>
     <form class="registration-form" [formGroup]="registerForm" action="" (submit)="registerUser()">
         <p class="input-label">Workspace name</p>
-        <mat-form-field class="input-form" appearance="fill" color="warn" floatLabel="always" hideRequiredMarker="true">
-            <input matInput placeholder="Name" formControlName="workspaceName" required>
+        <mat-form-field class="input-form" appearance="fill" floatLabel="always" hideRequiredMarker="true">
+            <input matInput placeholder="Name" formControlName="workspaceName" id="workspaceName">
         </mat-form-field>
+        <div class="errors">
+            <small *ngIf=" registerForm.controls['workspaceName'].touched && registerForm.controls['workspaceName'].errors && registerForm.controls['workspaceName'].errors['required']">
```

Comment:
```suggestion
            <small *ngIf=" registerForm.controls['workspaceName'].touched && registerForm.controls['workspaceName'].errors?.['required']">
```
The same everywhere below

Reply:
This has already been fixed